refinery/tool_manager/utils.py

```diff
@@ -282,11 +282,13 @@ def get_workflow_list():
     return workflow_list
 
 
-def mock_get_workflow_list():
+def mock_get_workflow_list(filename):
```

Comment:
This function simply reads a JSON file content and loads it into a Python object, so I'd name it accordingly and make it re-useable by allowing to specify any file path.